packages/trader/src/App/Components/Form/__tests__/number-selector.spec.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import NumberSelector from '../number-selector';
+
+const tests_arr_arr_numbers = [
+    [1, 2, 3],
+    [4, 5, 6],
+];
+const mock_props = {
+    arr_arr_numbers: tests_arr_arr_numbers,
```

Comment:
`arr_arr_numbers` was used for the prop of the component before I started to work on TS migration and testing. I decided to leave this name because it is partly self-declared.